### PR TITLE
fix: skia image size

### PR DESCRIPTION
**Pull request by HaakonSvane**
- Set default ImageMarkup draw color to red
- Fix ImageMarkup layout issues
- Remove MadLegacyComponents
- Fix typography issue with color prop not being passed to the theme engine

### CODE REVIEW

File: packages/skia-draw/src/premades/ImageMarkup/EDSControlPanel.tsx [hunk 1]
```diff
@@ -139,14 +139,14 @@ export const EDSControlPanel = (props: {
                 key={"undo-button"}
                 style={styles.buttonStyle}
             >
-                <Typography color="gray">undo</Typography>
+                <Typography color="secondary">undo</Typography>
```
Comment (hathibelagal-dev2) [inline packages/skia-draw/src/premades/ImageMarkup/EDSControlPanel.tsx:142]: Typography's color field I think should not be used with secondary/danger etc. In EDS, this field accepts HTML colors(hex values and named colors like blue, salmon, tomato etc)

File: packages/skia-draw/src/premades/ImageMarkup/EDSControlPanel.tsx [hunk 2]
```diff
@@ -139,14 +139,14 @@ export const EDSControlPanel = (props: {
...
             </PressableHighlight>
             <PressableHighlight
                 onPress={() => onPressClear()}
                 key={"clear-button"}
                 style={styles.buttonStyle}
             >
-                <Typography color="red">clear</Typography>
+                <Typography color="danger">clear</Typography>
```
Comment (hathibelagal-dev2) [inline packages/skia-draw/src/premades/ImageMarkup/EDSControlPanel.tsx:149]: Same as above

File: packages/skia-draw/src/Canvas.tsx [hunk 3]
```diff
@@ -23,42 +23,42 @@ const CanvasComponent: ForwardRefRenderFunction<
...
-                    />
-                ))}
-            {renderChildrenOnTop && children}
-        </SkiaCanvas>
-    );
-};
+        return (
+            <SkiaCanvas
+                ref={skiaCanvasRef}
+                style={StyleSheet.flatten([
```
Comment (hathibelagal-dev2) [inline packages/skia-draw/src/Canvas.tsx:37]: Is this flattening necessary?

**Review by hathibelagal-dev2 — CHANGES_REQUESTED** [review]
Cool, it's becoming a better library!

Might also want to remove the ignore pattern in the eslint-config package.

Is it possible to collapse/move the toolbar? It obscures a part of the image
simulator_screenshot_4606F748-F979-4D5F-B782-7FEF9D896655

File: packages/skia-draw/src/premades/ImageMarkup/EDSControlPanel.tsx (hunk 1 above)
Comment (HaakonSvane) [reply packages/skia-draw/src/premades/ImageMarkup/EDSControlPanel.tsx:142]: It seems Typography in EDS supports both Hex codes and shorthand codes like this (primary, secondary, danger, warning, etc..). You can verify this in the [eds storybook](https://eds-storybook-react.azurewebsites.net/?path=/docs/typography-typography--introduction#typography)
I'll add support for hex codes as well 👌

File: packages/skia-draw/src/Canvas.tsx (hunk 3 above)
Comment (HaakonSvane) [reply packages/skia-draw/src/Canvas.tsx]: Removed in ca9a088

File: packages/skia-draw/src/premades/ImageMarkup/EDSControlPanel.tsx (hunk 1 above)
Comment (HaakonSvane) [reply packages/skia-draw/src/premades/ImageMarkup/EDSControlPanel.tsx:142]: Fixed in ca9a088

**Review by hathibelagal-dev2 — APPROVED** [review]
Cool

**Pull request merged**